webtau-http/src/test/java/com/twosigma/webtau/http/HttpTestDataServer.java

```diff
@@ -67,7 +67,22 @@
         testServer.registerDelete("/resource", new TestServerTextResponse("abc"));
         testServer.registerGet("/params?a=1&b=text", new TestServerJsonResponse("{\"a\": 1, \"b\": \"text\"}"));
 
-        testServer.registerGet("/redirect", new TestServerRedirectResponse(HttpURLConnection.HTTP_MOVED_TEMP, testServer, "/end-point"));
+        testServer.registerGet("/redirect", new TestServerRedirectResponse(HttpURLConnection.HTTP_MOVED_TEMP, testServer, "/redirect2"));
```

Comment:
nit: since all the register redirect are related, should they be moved to a separate method?